bridge.go

```diff
@@ -101,6 +103,17 @@ func (b *Bridge) pairAs(appName string) error {
 	if err != nil {
 		return err
 	}
+
```

Comment:
> You're absolutely right, I've checked the documentation and the total limit is indeed 40 characters. Additionally, it is also specified that the application_name is maximum 20 chars and the devicename is 19.

Great! I tried to find that documentation and couldn't find it. No doc detailed enough.

I updated this PR